lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -157,10 +159,55 @@ class Meta:
             "gradeStatus",
             "lockStatus",
             "score",
+            "email",
+            "fullname",
         ]
         read_only_fields = fields
 
 
+class AssessmentScoresSerializer(serializers.Serializer):
+    """
+    Assessment Score for displaying assessment table in Enhanced Staff Grader (ESG)
+    """
+    criterion_name = serializers.CharField()
+    score_earned = serializers.IntegerField()
+    score_type = serializers.CharField()
+
+    class Meta:
+        fields = [
+            "criterion_name",
+            "score_earned",
+            "score_type",
+        ]
+        read_only_fields = fields
+
+
+class AssessmentSerializer(serializers.Serializer):
+    """
+    Assessment metadata for displaying assessment table in Enhanced Staff Grader (ESG)
+    """
```

Comment:
Is this and this serializer the same? https://github.com/eduNEXT/edx-platform/pull/298/files#diff-529906c051f19c6b9672b9d9634be578fa4e9681d392ea5632443d94ef174b46R169-R171 their docstring is pretty similar